packages/office-ui-fabric-react/src/components/MessageBar/MessageBar.scss

```diff
@@ -106,10 +107,13 @@
   }
 }
 
-// Shared
+// Add space between adjacent MessageBars.
+.root + .root {
+  margin-top: 6px;
+}
```

Comment:
we're trying to avoid this style of spacing (and will be removing it from buttons and other components soon). A component should have zero opinion of the space around it. For now the implimentor will need to add that margin, and in the future we'll have layout components to add space between children.